.eslintrc

```diff
@@ -5,28 +5,37 @@
         "node": true
     },
     "plugins": [
-        "@typescript-eslint",
-        "svelte3"
+        "@typescript-eslint"
     ],
     "overrides": [
         {
             "files": [
                 "*.svelte"
             ],
-            "processor": "svelte3/svelte3"
+            "parser": "svelte-eslint-parser",
+            "parserOptions": {
+                "parser": "@typescript-eslint/parser"
+            },
+            "rules": {
+                "no-undef": "off" // TODO: remove this line
```

Comment:
Here I temporarily suppressing the `no-undef` errors, which currently occurs for global objects such as `window`, `app` and so on. I can't solve this problem right now, so this is a temporary work around.